spec/controllers/content_items_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe ContentItemsController, type: :controller do
+  describe "GET #index" do
+    let(:organisation) { FactoryGirl.create(:organisation) }
+
+    before do
+      get :index, params: { organisation_id: organisation }
+    end
+
+    it "returns http success" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "assigns content_items" do
```

Comment:
I can see your point for stubbing this out this way.. in the organisation's factory girl definition 

we already cater for this.. 

```
let(:organisation) { FactoryGirl.create(:organisation_with_content_items, content_items_count: 1) }
```

From here the only difference is that create would return a saved instance